extensions/hibernate-orm/deployment/pom.xml

```diff
@@ -63,6 +63,11 @@
             <optional>true</optional>  <!-- conditional dependency -->
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-devui-deployment</artifactId>
+            <optional>true</optional>
+        </dependency>
```

Comment:
I suppose at some point, we should move the Dev UI stuff of this extension to a separate artifact and then we could drop this dependency?

Reply:
Ideally, yes.

Reply:
One of my PRs already removed it. It does not seem to be used.

Reply:
We have a concept of an extension depending on another extension though. It's misaligned now but in this specific case it appears to be ok.